openverse_catalog/dags/providers/provider_csv_load_scripts/inaturalist/01_photos.sql

```diff
@@ -0,0 +1,39 @@
+/* 
+********************************************************************************
+PHOTOS 
+********************************************************************************
+--  not using an FK constraint on observer_id to save load time, but the 5/30/2022 dataset does have complete observer ids
+--  not using an FK constraint on observation_uuid to save load time, but the 5/30/2022 dataset does have complete observer ids
+--  photo_id is not unique. There are ~130,000 photo_ids that appear more than once, maybe because an earlier version of 
+    the photo was deleted (?) unclear, but for now I'm going to assume that they will be taken care of later in the processing 
+    of these data. It does mean that we can't add an index and things will go slower when selecting on photo id. 
```

Comment:
I should have said unique / pk index here.

Reply:
If we do use the same tsv -> `load_data` steps as our other providers, we should [handle duplicate foreign ids](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/common/loader/sql.py#L178), although I'm curious if there's a preferred way to select the record that is kept depending on how the duplicates arise (I imagine that would be out of scope here).

Reply:
It's on the order of 0.1% of the records, and I didn't see anything in [the documentation](https://github.com/inaturalist/inaturalist-open-data) about recommended handling. I do think we could improve data quality somewhat by comparing the tsv data against the results from `s3 ls s3://inaturalist-open-data/photos` but I wonder if that could be a separate improvement task.